CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.4.4 (March 6, 2024)
```

Comment:
Can you remove this section for now?  It will be added when we are preparing the release.

Reply:
Done